moveit_kinematics/kdl_kinematics_plugin/include/moveit/kdl_kinematics_plugin/chainiksolver_vel_mimic_svd.hpp

```diff
@@ -79,6 +79,7 @@ class ChainIkSolverVelMimicSVD : public ChainIkSolverVel
                      Eigen::Matrix<double, 6, 1>::Constant(1.0));
   }
 
+  // NOLINTNEXTLINE(readability-identifier-naming)
```

Comment:
[nit] why aren't we following our own naming rules.  It seems like there are are an excessive amount of these NOLINT comments.

Reply:
This follows the naming in KDL...

Reply:
This specific `NOLINTNEXTLINE` does not work, likely because of the comment in between.
See CI for `master`: https://travis-ci.org/github/ros-planning/moveit/jobs/683731531

Reply:
Damn it. Why did CI pass here? Please go ahead and fix this in your PR.